src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.WriterService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriterServiceImplTest {
+    private static final String PATH_TO_FILE = "src/test/resources/report.csv";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String INVALID_PATH = "invalid/path/to/file";
+    private static WriterService writerService;
+
+    @BeforeClass
+    public static void setUp() {
+        writerService = new WriterServiceImpl();
+    }
+
+    @Test
+    public void write_To_File_Ok() throws IOException {

```

Comment:
let's organize the thest in 2 sections split by empty line:

1. initialize all expected and actual variables
2. perform assertions

Reply:
ok. )) I did it)